includes/internal/emails/class-email-preview.php

```diff
@@ -217,10 +219,27 @@ private function validate_request(): void {
 			wp_die( esc_html__( 'Invalid request', 'sensei-lms' ) );
 		}
 
+		// phpcs:ignore WordPress.WP.Capabilities.Unknown
 		if ( ! current_user_can( 'manage_sensei' ) ) {
 			wp_die( esc_html__( 'Insufficient permissions', 'sensei-lms' ) );
 		}
 
 		check_admin_referer( 'preview-email-post_' . $post->ID );
 	}
+
+	/**
+	 * Filter the preview link.
+	 *
+	 * @param string  $link The preview link.
+	 * @param WP_Post $post The post object.
+	 *
+	 * @return string
+	 */
+	public function filter_preview_link( $link, $post ): string {
+		if ( Email_Post_Type::POST_TYPE !== $post->post_type ) {
+			return $link;
+		}
+
+		return str_replace( '&amp;', '&', $this->get_preview_link( $post->ID ) );
```

Comment:
Our `get_preview_link` uses `wp_nonce_url` that encodes the url, but it is encoded one more time on the frontend, so I had to remove encoded entity here.
But I'm going to replace it with the usage of `wp_create_nonce` instead, at least here.

Reply:
Left it as is: the original method is used in many places, and I don't won't duplicate its logic.